app/component/app-gallery/app-gallery.html

```diff
@@ -0,0 +1,19 @@
+<section class = 'app-gallery'>
+
+  <form novalidate ng-submit = "appGalleryCtrl.addGalleryItem()">
+    <label for = "title">title:<label>
```

Comment:
It's usually considered a best practice not to have spaces in your html between the attributes and their values.